ckan/logic/action/get.py

```diff
@@ -2546,7 +2547,9 @@ def package_activity_list(context, data_dict):
     '''
     # FIXME: Filter out activities whose subject or object the user is not
     # authorized to read.
-    _check_access('package_show', context, data_dict)
+    data_dict['include_data'] = False
+    include_hidden_activity = asbool(context.get('include_hidden_activity'))
```

Comment:
Why not make `include_hidden_activity` part of the data_dict instead?

Reply:
'hidden' activity is that by site_user, such as harvests, scripts, which are not shown in the activity stream because they can be too numerous and aren't attributed to a person. This is not something this PR changes. (I guess you could argue they should all be exposed, but I didn't want to change anything unnecessarily.)

However we do need to get at these hidden activities in the migration, because these hidden activities do have Activity objects, and if a hidden Activity is followed be a non-hidden one and you look at the changes of that non-hidden Activity, then it does a diff with the hidden one (rather than the most recent non-hidden one), so it is important to store the package_dict in hidden Activity objects.

So because we don't want to start exposing these activities in the Activity Stream web page or API, it's added as an option on the context, not the data_dict.

Reply:
what if `include_hidden_activity` was an option in the data_dict but only available to sysadmin users, i.e with the `ignore_not_sysadmin` validator? Adding options to the context means those calls are limited to a user running locally, but I think a sysadmin should be able to access the same things from an API whether local or remote.

Reply:
Nice idea. I've done this now in 75592d95b